tRecorderApi/htmlcov/api_migrations_0001_initial_py.html

```diff
@@ -150,11 +150,12 @@ <h2 class="stats">
 <p id="n81" class="pln"><a href="#n81">81</a></p>
```

Comment:
Also look at the other files in tRecorderApi/htmlcov/ to see if there are other runtime generated files that can be gitignored